scripts/deployment/main.py

```diff
@@ -67,7 +68,18 @@ def contract_to_json(contracts: dict[str, DeployedContract]) -> dict:
     }
 
 
-def deploy_contract(web3: Web3, name: str, *args: Any) -> DeployedContract:
+def deploy_contract(web3: Web3, constructor_spec: Union[str, Sequence]) -> DeployedContract:
+    # constructor_spec is either
```

Comment:
If this is so, then the type annotation is wrong.

Reply:
> If this is so, then the type annotation can be improved

I'm not sure what you mean, could you please expand?
If you meant something like

```python
constructor_spec: Union[str, Sequence[str]]
```

that would unfortunately not work because there are types other than `str` that may appear there.
Maybe something like `Union[str, Sequence[Union[str, int, ChecksumAddress]]]` might work, but I haven't tried...